### PR TITLE
Added robots.txt and added sponsorship page

### PR DESCRIPTION
**Pull request by whitef0x0**
Added sponsorship page with tiers as well as a robots.txt to prevent google from crawling the sponsorship page.

### CODE REVIEW

**Comment by chan-kelv** [comment]
@arnoldlam @jordanschalm @arabelle

**Comment by jordanschalm** [comment]
There are some typos in the sponsorship PDF i.e. "UBC Launchpad is a Engineering Design Team at UBC that focuses on **build** web and mobile apps using proper Software Engineering techniques"

**Comment by jordanschalm** [comment]
Title should have space between "Launch" and "Pad"

File: sponsors.html [hunk 1]
```diff
@@ -0,0 +1,263 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Sponsor Us | UBC LaunchPad</title>
```
Comment (jordanschalm) [inline sponsors.html:4]: Title should have space between "Launch" and "Pad"

File: sponsors.html [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+	<link href='https://fonts.googleapis.com/css?family=Lato:300,400,700,900' rel='stylesheet' type='text/css'>
+	<link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css" rel="stylesheet" type="text/css">
+	<link rel="stylesheet" href="css/stylesheets/site.css">	
+	<link rel="stylesheet" href="css/stylesheets/sponsors.css">
+	<link rel="stylesheet" href="css/fontello.css">
```
Comment (jordanschalm) [inline sponsors.html:9]: `fontello.css` file doesn't exist

File: sponsors.html [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+	<link rel="icon" type="image/png" href="favicon.png">
+	
+	<meta
+		name="viewport"
+		content="
+      width=device-width,
+      initial-scale=1,
+      minimum-scale=1,
+      maximum-scale=1
+    "
```
Comment (jordanschalm) [inline sponsors.html:19]: Can we put this tag on one line, causes errors

File: sponsors.html [hunk 4]
```diff
@@ -0,0 +1,263 @@
...
+        </ul>
+      </div>
+    </div>
+
+<header class="sponsor-header">
+	<div id="hero" class="sponsor-hero"><h2>Sponsor us.</h2></div>
+</header>
+<section id="about">
+	<article class="grid-container grid-parent">
+		<h2 class="grid-100">Overview</h2>
```
Comment (jordanschalm) [inline sponsors.html:53]: In this section, can we use the same h1, h2 and p styles as similar sections on the homepage

**Pull request merged**